### DISCUSSION TITLE
[help] Get atom of atomFamily while id of it is choosen at runtime

### DISCUSSION THREAD

**Discussion by ChrisCross3** [facebookexperimental/Recoil#1475] [Q&A]
I want to output the value of an atom from its family in a function "renderAtomFamilyAtomValue".
The id of the atom is selected at runtime by the user. As I understand it, I can't do that, because the id has to be specified on initialization the hook and can't specify it dynamically in a function somewhere.
Is there a way to do it?

**Comment by drarmstr** [comment]
While React hooks have to have a constant order during rendering, which atom you might use with it can be determined dynamically.  So, you can select the atom to use at runtime from user input or other state.

**Comment by ChrisCross3** [comment]
Thank you for your answer @drarmstr.
Is there an example code somewhere or can you give one?
